Packs/iDefense/Integrations/iDefense_v2/iDefense_v2.yml

```diff
@@ -0,0 +1,172 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: iDefense_v2
+  version: -1
+configuration:
+- display: URL
+  name: url
+  defaultvalue: https://api.intelgraph.idefense.com/
+  type: 0
+  required: true
+- display: API Token
+  name: api_token
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: use_proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+description: IDefense provides intelligence regarding security threats and vulnerabilities.
+display: iDefense v2
+name: iDefense_v2
+script:
+  commands:
+  - name: ip
+    arguments:
+    - name: ip
```

Comment:
i think you should support an array and handle it for the code, same for all other reputation commands(domain, url). @DeanArbel correct me if i'm wrong.